### PR TITLE
feat: add credential columns in admin ui for omise keys

### LINKED ISSUE #2860: Add Omise Keys column in Admin UI
Add Omise Key Columns similar to paypal credentials

- Add mode, test and live keys (public and secret)

Parent Issue- #2859 
**Additional context**

Working on it

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds appropriate columns and integrates them in settings module for omise keys

#### Changes proposed in this pull request:

- provide seperate options for test and live mode
- add appropriate columns in the setting model
- apply validations in the form component

Fixes #2860

### CODE REVIEW

**Comment by uds5501** [comment]
@CosmicCoder96 @ritikamotwani @kushthedude @mariobehling Have a look?

image

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 1]
```diff
@@ -103,6 +103,74 @@
...
+                 name='omise_integration_mode'
+                 value='live'
+                 current=settings.omiseMode
+                 onChange=(action (mut settings.omiseMode))}}
+    </div>
+    {{#if (eq settings.omiseMode 'live')}}
+      <div class="field">
+        <label>
+          {{t 'Omise Live Public Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This pucliv key will be used during live (production) mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: use translation helpers for `content`.

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 2]
```diff
@@ -103,6 +103,74 @@
...
+        </label>
+        {{input type='text' name='live_public_key' value=settings.omiseLivePublic}}
+      </div>
+      <div class="field">
+        <label>
+          {{t 'Omise Live Secret Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This secret key will be used during live (production) mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: use translation helpers for content.

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 3]
```diff
@@ -103,6 +103,74 @@
...
+        <label>
+          {{t 'Omise Test Public Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This public key will be used for testing mode"}}
+        </label>
+        {{input type='text' name='test_public_key' value=settings.omiseTestPublic}}
+      </div>
+      <div class="field">
+        <label>
+          {{t 'Omise Test Secret Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This secret key will be used during testing mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: use translation helpers for content.

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 4]
```diff
@@ -103,6 +103,74 @@
...
+                 name='omise_integration_mode'
+                 value='test'
+                 current=settings.omiseMode
+                 onChange=(action (mut settings.omiseMode))}}
+    </div>
+    {{#if (eq settings.omiseMode 'test')}}
+      <div class="field">
+        <label>
+          {{t 'Omise Test Public Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This public key will be used for testing mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: use translation helpers for content.

**Comment by uds5501** [comment]
@CosmicCoder96 Done. Please review

**Pull request merged**